packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,28 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
```

Comment:
Don't remove width/height. It worries me to leave it without explicit dimensions. (You can use border-box:box-sizing to have border dimensions included in the base dimensions, instead of in addition to.)